src/request/export/ExportWords.js

```diff
@@ -128,105 +148,23 @@ class ExportWords extends Nimiq.Observable {
-        window.location.hash = ExportWords.Pages.SHOW_WORDS;
-    }
-
-    _finish() {
-        const result = {
-            success: true,
-        };
-        this._resolve(result);
-    }
-
-    _buildPrivacy() {
-        const $el = document.createElement('div');
-        $el.id = ExportWords.Pages.PRIVACY;
-        $el.classList.add('page', 'nq-card');
-        $el.innerHTML = `
-        <div class="page-header nq-card-header">
-            <a tabindex="0" class="page-header-back-button nq-icon arrow-left"></a>
-            <h1 data-i18n="recovery-words-title" class="nq-h1">Recovery Words</h1>
-        </div>
-
-        <div class="page-body nq-card-body">
-            <div class="privacy-agent">
-                <div class="privacy-warning"></div>
-            </div>
-            <div class="flex-grow"></div>
-        </div>
-
-        <div class="page-footer">
-            <button data-i18n="recovery-words-continue-to-words" class="nq-button">Continue to Recovery Words</button>
-            <form class="passphrase-box hide-if-key-active"></form>
-        </div>
-        `;
-        /** @type {HTMLElement} */
-        const $app = (document.getElementById('app'));
-        $app.insertBefore($el, $app.children[1]);
-        return $el;
-    }
-
-    _buildRecoveryWords() {
-        const $el = document.createElement('div');
-        $el.id = ExportWords.Pages.SHOW_WORDS;
-        $el.classList.add('page', 'nq-card');
-        $el.innerHTML = `
-        <div class="page-header nq-card-header">
-            <a tabindex="0" class="page-header-back-button nq-icon arrow-left"></a>
-            <h1 data-i18n="recovery-words-title" class="nq-h1">Recovery Words</h1>
-        </div>
-
-        <div class="page-body nq-card-body">
-            <div class="recovery-words"></div>
-        </div>
-
-        <div class="page-footer">
-            <button class="to-validate-words nq-button" data-i18n="continue">Continue</button>
-        </div>
-        `;
-        /** @type {HTMLElement} */
-        const $app = (document.getElementById('app'));
-        $app.insertBefore($el, $app.children[1]);
-        return $el;
-    }
-
-    _buildValidateWords() {
-        const $el = document.createElement('div');
-        $el.id = ExportWords.Pages.VALIDATE_WORDS;
-        $el.classList.add('page', 'nq-card');
-        $el.innerHTML = `
-        <div class="page-header nq-card-header">
-            <a tabindex="0" class="page-header-back-button nq-icon arrow-left"></a>
-            <h1 data-i18n="create-heading-validate-backup" class="nq-h1">Validate your backup</h1>
-        </div>
-
-        <div class="page-body nq-card-body">
-            <div class="validate-words"></div>
-        </div>
-        `;
-        /** @type {HTMLElement} */
-        const $app = (document.getElementById('app'));
-        $app.insertBefore($el, $app.children[1]);
-        return $el;
+        this._$noRecovery.classList.toggle('key-active', this._key !== null);
```

Comment:
What does `key-active` stand for and do? The classname is not descriptive enough.